code-of-conduct.md

```diff
@@ -1 +1,76 @@
 # Triangle DevOps Code of Conduct
+
+Triangle DevOps is dedicated to providing a harassment-free community experience
+for everyone, regardless of gender, gender identity and expression, sexual
+orientation, disability, physical appearance, body size, race, age or religion.
+We do not tolerate harassment of community participants in any form.
+
+Community participants violating these rules may be sanctioned or expelled from
```

Comment:
One thing that jumps out at me is that it's not clear that this code of conduct also applies to the organizers. I of course expect that it does, but we should make that clear and make clear that organizers will be treated no differently from anyone else.